build/build.cake

```diff
@@ -203,19 +203,23 @@ Task("Version")
     Information($"\r\nBuild Version: {Version}");
 });
 
+Task("Restore")
+    .Description("Restore all Nuget packages used by solution")
+    .IsDependentOn("Version")
```

Comment:
Why does it depend on Version?

Reply:
I still don't get it... why does it depend on Clean?

---
In reply to: [280585318](https://github.com/windows-toolkit/SceneLoader/pull/1#discussion_r280585318) [](ancestors = 280585318)

Reply:
I started running into a build error saying msbuild is being used by a different process when I removed it. 

---
In reply to: [280957800](https://github.com/windows-toolkit/SceneLoader/pull/1#discussion_r280957800) [](ancestors = 280957800,280585318)